tests/shell/BashateBearTest.py

```diff
@@ -0,0 +1,28 @@
+from bears.shell.BashateBear import BashateBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+E002_bad = """
+  echo "E002: Has a tab"
+"""
+
+E003_bad = """
+   indent is not multiple of 4
+"""
+long_line = 'a = "{0}"'.format('a' * 100)
```

Comment:
It seems that this is the E006 error. For consistency I would maybe call the variable `E006_bad` (and add a comment that this means "too long lines". You could add such comments also for the two other variables explaining the meaning of the error codes.)